src/Components/AllReviews.jsx

```diff
@@ -1,12 +1,15 @@
 import { useEffect, useState } from "react";
 import { getReviews } from "../Utils/api";
 import ReviewCards from "./ReviewCards";
+import { useSearchParams } from "react-router-dom";
 
 export default function AllReviews() {
   const [reviews, setReviews] = useState([]);
+  const [searchParams, setSearchParams] = useSearchParams();
+  const category = searchParams.get("category");
 
   useEffect(() => {
-    getReviews().then((reviews) => {
+    getReviews(category).then((reviews) => {
       setReviews(reviews);
     });
   }, []);
```

Comment:
you need categories in the dependancy array, react is throwing a warning, these shouldn't be ignored.

Reply:
done =)